proxy/http/HttpTransact.cc

```diff
@@ -693,6 +693,8 @@ HttpTransact::EndRemapRequest(State *s)
     switch (s->http_return_code) {
     case HTTP_STATUS_MOVED_PERMANENTLY:
     case HTTP_STATUS_PERMANENT_REDIRECT:
+    case HTTP_STATUS_SEE_OTHER:
+    case HTTP_STATUS_USE_PROXY:
```

Comment:
Don't we want / need HTTP_STATUS_TEMPORARY_REDIRECT here too ?

Reply:
It is there, but in the next case along with `HTTP_STATUS_MOVED_TEMPORARILY`. These want the template `redirect#moved_temporarily`.

Reply:
The point being that we don't have to modify this when they add new redirect responses. But if we feel strongly, we can restrict it. Not sure it's worth that hassle though, since it's in a plugin, and people can shoot themselves in many other ways if they like to :-) [This check is at configuration parse time].